fabric-biome-api-v1/src/main/java/net/fabricmc/fabric/api/biome/v1/BiomeModification.java

```diff
@@ -17,23 +17,22 @@
 
 package net.fabricmc.fabric.api.biome.v1;
 
-import java.util.function.BiConsumer;
 import java.util.function.Consumer;
 import java.util.function.Predicate;
 
 import org.jetbrains.annotations.ApiStatus;
 
 import net.minecraft.util.Identifier;
 
-import net.fabricmc.fabric.impl.biome.modification.BiomeModificationImpl;
-
 /**
  * <b>Experimental feature</b>, may be removed or changed without further notice.
  *
  * @see BiomeModifications
  */
+@Deprecated
 public class BiomeModification {
 	private final Identifier id;
+	public static final org.quiltmc.qsl.worldgen.biome.impl.modification.BiomeModificationImpl QUILT_INSTANCE = org.quiltmc.qsl.worldgen.biome.impl.modification.BiomeModificationImpl.INSTANCE;
```

Comment:
Kinda nitpicky, but can't a lot of the stuff in this class be imported? Would be a lot cleaner to read & maintain

Reply:
A lot of places it's because Fabric also has the same class name, but I think some places Fabric isn't imported. Let me go through and check